examples/cloud_run_http/README.md

```diff
@@ -29,11 +29,8 @@ COPY . .
 RUN pip install gunicorn functions-framework
```

Comment:
We can also remove `gunicorn` here as it's now a sub-dependency of `functions-framework`.

Reply:
Updated